twake/frontend/src/app/scenes/Client/ChannelsBar/Parts/Channel/ChannelIntermediate.tsx

```diff
@@ -64,6 +71,7 @@ export default (props: Props): JSX.Element => {
       visibility={channel.data.visibility || 'public'}
       notifications={notifications.length || 0}
       menu={menu(channel)}
+      menuOpened={menuOpened}
```

Comment:
I definitively prefer "active" instead of "menuOpened"

Reply:
Done 👍🏻